src/wp-includes/css/custom-properties.css

```diff
@@ -92,6 +92,16 @@ body {
 
 	--wp-admin--plugin-modal-banner-title--color: #fff;
 	--wp-admin--plugin-modal-banner-title--background: rgba(29, 35, 39, 0.9);
+
+	--wp-admin--login-anchor--color--hover: #135e96;
+	--wp-admin--login-anchor--color--focus: #043959;
+	--wp-admin--login-anchor--box-shadow-from--focus: #4f94d4;
+	--wp-admin--login-anchor--box-shadow-to--focus: rgba(79, 148, 212, 0.8);
```

Comment:
Box shadows are tricky… I'm wondering if there's a better way to handle this combination of multiple size/style/color properties. In the main PR I tried to standardize on this for focus styles, but I could also see this whole value being a custom property (`--wp-admin--??--focus` ?).

```css
box-shadow:
		0 0 0 1px var(--wp-admin--button-primary--hover),
		0 0 2px 1px var(--wp-admin--button-primary);
```

In other places, I used the `--wp-admin--shadow` property with [functional `rgb` notation](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb()), which avoids needing to define the alpha value - instead there's one property, `--wp-admin--shadow`, which is `0 0 0`. I'm on the fence about whether this is a stop-gap measure until we audit the opacity values, or if this is a good way to reduce excess properties.

```css
box-shadow: 0 1px 1px rgb(var(--wp-admin--shadow) / 0.04);
```